src/collections/_documentation/development/sdk-dev/event-payloads/breadcrumbs.md

```diff
@@ -3,14 +3,23 @@ title: Breadcrumbs Interface
 sidebar_order: 6
 ---
 
-The breadcrumbs interface specifies a series of application events, or
-`breadcrumbs`, that occurred before the main event. Its canonical name is
-`breadcrumbs`.
+The Breadcrumbs Interface specifies a series of application events, or
+"breadcrumbs", that occurred before and led up to an event.
 
-This interface is an object with a single `values` key containing an ordered
-list of breadcrumb objects. The entries are ordered from oldest to newest.
-Consequently, the last entry in the array should be the last entry before the
-event occurred.
+An [event]({%- link _documentation/development/sdk-dev/event-payloads/index.md
+-%}) may contain one or more breadcrumbs in an attribute named `breadcrumbs`.
+
+This interface is an object with a single `values` attribute containing an
+ordered list of breadcrumb objects. The entries are ordered from oldest to
+newest. Consequently, the last entry in the list should be the last entry before
+the event occurred.
+
+{% capture __alert_content -%}
+For backwards compatibility, the server also understands events with exceptions
+that are not in the canonical format described here.
```

Comment:
This talks about exceptions - copy/paste error?

Reply:
And removed preferring a more global note, as per Jan's suggestion.